src/helm-chart/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v2
+name: papermc-server
+description: A Helm chart for deploying a PaperMC server
+version: 0.0.1
```

Comment:
The version is always overridden at delivering time. Therefore, just put a version like `0.0.0-DEV` and that's all.